src/components/http-error-message/test/e2e/http-error-message.js

```diff
@@ -0,0 +1,59 @@
+import { VlElement, By } from '../../../../utils/test';
+import { VlTypography } from '../../../typography/test/e2e/typography';
+import { VlButton } from '../../../button/test/e2e/button';
+
+export default class VlHttpErrorMessage extends VlElement {
+  async getTitle() {
+    return (await this._getTitle()).getText();
+  }
+
+  async getContent() {
+    const typography = await this._getTypography();
+    const slot = await typography.shadowRoot.findElement(By.css(`slot[name="text"]`));
+    const elements = await typography.getAssignedElements(slot);
+    const secondSlotElements = await this.getAssignedElements(elements[0]);
+    return secondSlotElements[0];
```

Comment:
Ik zou de array destructuren, is iets leesbaarder en een mooiere oplossing. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment